### DISCUSSION TITLE
Using AWS Managed Prometheus Workspace

### DISCUSSION THREAD

**Discussion by SteelAlloy** [aws-ia/terraform-aws-eks-blueprints-addons#193] [Q&A]
Hi,

Previously (blueprints v4) AMP Workspaces were available to use with the prometheus addon.

```yaml
  # Creates the AMP workspace and all the relevant IAM Roles
  enable_amazon_prometheus = true

  # Deploys Prometheus server with remote write to AWS AMP Workspace
  enable_prometheus = true
```

After digging a bit, it was removed in v5 in favor of `kube_prometheus_stack`.

https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/pull/130/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbL2511

```yaml
enable_kube_prometheus_stack        = true

# no more AMP
```

Is there any simple solution to link an AMP Workspace to this stack now ? The new `kube_prometheus_stack` doesn't appear to support such feature

**Comment by askulkarni2** [comment]
Hi @oganexon, thanks for the issue. We decided not to try and do this ourselves and instead leave it to the experts to provide that guidance. We have provided kube-prometheus-stack here because it allows you setup a monitoring stack on cluster quickly. But for detailed setup, please take a look at https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main. It is a collection of opinionated modules for AMP, AMG, ADOT, etc. Please provide any feedback you may have on that repo.
